irohad/torii/impl/command_service.cpp

```diff
@@ -69,6 +64,16 @@ namespace torii {
     });
   }
 
+  namespace {
+    iroha::protocol::ToriiResponse makeResponse(
+        shared_model::crypto::Hash h, iroha::protocol::TxStatus status) {
+      iroha::protocol::ToriiResponse response;
```

Comment:
Why are we using raw proto instead of the model?

Reply:
Because it used for grpc